man/sd_event_source_set_exit_on_failure.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="sd_event_source_set_exit_on_failure" xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>sd_event_source_set_exit_on_failure</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_event_source_set_exit_on_failure</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_event_source_set_exit_on_failure</refname>
+    <refname>sd_event_source_get_exit_on_failure</refname>
+
+    <refpurpose>Set or retrieve the exit on failure feature of event sources</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-event.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int <function>sd_event_source_set_exit_on_failure</function></funcdef>
+        <paramdef>sd_event_source *<parameter>source</parameter></paramdef>
+        <paramdef>int <parameter>b</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_event_source_get_exit_on_failure</function></funcdef>
+        <paramdef>sd_event_source *<parameter>source</parameter></paramdef>
+      </funcprototype>
+
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_event_source_set_exit_on_failure()</function> may be used to set/unset the
+    exit-on-failure flag of the event source object specified as <parameter>source</parameter>. The flag
+    defaults to off. If on and the callback function set for the event source returns a failure code (i.e. a
+    negative value) the event loop is exited too, using the callback return code as exit code for
+    <citerefentry><refentrytitle>sd_event_exit</refentrytitle><manvolnum>3</manvolnum></citerefentry>. If off
+    the event source is disabled but the event loop continues to run. Setting this flag is useful for
+    "dominant" event sources, that define the purpose and reason for the event loop, and whose failure hence
+    should propagate to the event loop itself — as opposed to "auxiliary" event sources whose failures should
+    remain local and affect the event source, but not propagate further.</para>
+
+    <para><function>sd_event_source_get_exit_on_failure()</function> may be used to query the flag currently
+    set for the event source object <parameter>source</parameter>.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>Return Value</title>
+
+    <para>On success, <function>sd_event_source_set_exit_on_failure()</function> returns a non-negative
+    integer.  <function>sd_event_source_get_exit_on_failure()</function> returns 0 if the flag is off, &gt; 0
```

Comment:
```suggestion
    integer.  <function>sd_event_source_get_exit_on_failure()</function> returns 0 if the flag is off, &gt; 1
```

Reply:
no, it's supposed to say `> 0`, which in xml becomes `&gt; 0`. The idea here is to say positive, larger than zero. We actually return 1 here, but we don#t want to say this because apps should not bother too much with the exact value, but just check > 0.